### DISCUSSION TITLE
It stopped working?

### DISCUSSION THREAD

**Discussion by nothitcombo** [Discord-OTP-Forcer/Discord-OTP-Forcer#79] [Community Support]
I load up the software, I was using it the other day and it was working fine, but now it only puts in the password/email but wont do the 6 digit code?

**Comment by Inbydev** [comment]
@nothitcombo could you try with the latest commit?

https://github.com/Derpitron/Discord-OTP-Forcer/archive/refs/heads/main.zip

And follow the tutorial:

https://github.com/Derpitron/Discord-OTP-Forcer/wiki/How-to-setup-and-use

**Comment by Inbydev** [comment]
This probably is an Issue of the past, the program got a lot of updates since then, so re open this discussion if you encounter the same issue with the latest release!

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta

And follow the tutorial:

https://discord-otp-forcer.codeberg.page/en/
